pkg/bloomcompactor/bloomcompactor.go

```diff
@@ -344,16 +340,16 @@ func (c *Compactor) compactUsers(ctx context.Context, logger log.Logger, sc stor
 	// TODO: Delete local files for unowned tenants, if there are any.
 }
 
-func (c *Compactor) compactTenant(ctx context.Context, logger log.Logger, sc storeClient, tableName string, tenant string) error {
```

Comment:
Why are we removing the logger here? This logger is a child of c.logger but with the tenant key.
https://github.com/grafana/loki/pull/11285/files#diff-a88e1e7a49788f4b1dbe6ea4b03845217353b07f7cae25e0f74042b693f50680R285

Reply:
Ah I see. Gonna revert that!